Makefile

```diff
@@ -473,7 +473,7 @@ uninstall:
 # Linters --------------------------------------------- {{{1
 #
 
-PYTHON_SCRIPTS := utils/hare-shutdown utils/hare-status utils/gen-uuid utils/utils.py
+PYTHON_SCRIPTS := $(find utils/ -name 'hare-*' -exec file {} \;) utils/utils.py
```

Comment:
Several concerns:
1. Are we sure that `file` is available at the build machine? If we use it, then our hare.spec file must also be updated (add `file` package to `BuildRequires`)
2. Did you forget to filter out the files which are not Python ones?
3. During the meeting yesterday I showed another version of command - based on grep only (no `file` dependency).